tests/Doctrine/Tests/Models/CMS/CmsUser.php

```diff
@@ -454,7 +454,7 @@ public static function loadMetadata(ClassMetadata $metadata): void
                             ],
                             [
                                 'name'      => 'phonenumbers.phonenumber',
-                                'column'    => 'number',
+                                'column'    => '`number`',
```

Comment:
Is this test about dealing with keywords? If not, would it make sense to use some other column name?

Reply:
I can't say. But if you use an alias for a reserved word column, there will be an error.